Find a functionally equivalent code:android.widget.TextView.onPreDraw:COMMENT
Method Modifier: public      
Comment:/**
 * {@inheritDoc}
 */

@@ -35,41 +35,40 @@
     if (this instanceof ExtractEditText && hasSelection() && mEditor != null) {
         mEditor.startSelectionActionMode();
     }
-    getViewTreeObserver().removeOnPreDrawListener(this);
-    mPreDrawRegistered = false;
+    unregisterForPreDraw();
     return true;
 }

